depthcharge/__init__.py

```diff
@@ -1,17 +1,28 @@
 """Initialize the depthcharge package."""
-from . import (
-    data,
-    encoders,
-    feedforward,
-    tokenizers,
-    transformers,
-)
-from .primitives import (
-    MassSpectrum,
-    Molecule,
-    Peptide,
-    PeptideIons,
-)
-from .version import _get_version
+# Ignore a bunch of pkg_resources warnings from dependencies:
+import warnings
+
+with warnings.catch_warnings():
```

Comment:
Have these deprecation warnings been reported to the psims people?

Reply:
Alas no, mostly because they originate from a variety of packages.

Reply:
```bash
python -X dev -c 'import psims'
```
yields:
```python
/Users/wfondrie/mambaforge/lib/python3.10/site-packages/pyteomics/mass/unimod.py:53: MovedIn20Warning: The ``declarative_base()`` function is now available as sqlalchemy.orm.declarative_base(). (deprecated since: 2.0) (Background on SQLAlchemy 2.0 at: https://sqlalche.me/e/b8d9)
  Base = declarative_base(metaclass=SubclassRegisteringDeclarativeMeta)
/Users/wfondrie/mambaforge/lib/python3.10/site-packages/psims/validation/validator.py:1: DeprecationWarning: pkg_resources is deprecated as an API. See https://setuptools.pypa.io/en/latest/pkg_resources.html
  import pkg_resources
/Users/wfondrie/mambaforge/lib/python3.10/site-packages/pkg_resources/__init__.py:2871: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('google')`.
Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
  declare_namespace(pkg)
/Users/wfondrie/mambaforge/lib/python3.10/site-packages/pkg_resources/__init__.py:2871: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('pyteomics')`.
Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
  declare_namespace(pkg)
/Users/wfondrie/mambaforge/lib/python3.10/site-packages/pkg_resources/__init__.py:2871: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('repoze')`.
Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
  declare_namespace(pkg)
/Users/wfondrie/mambaforge/lib/python3.10/site-packages/pkg_resources/__init__.py:2871: DeprecationWarning: Deprecated call to `pkg_resources.declare_namespace('sphinxcontrib')`.
Implementing implicit namespace packages (as specified in PEP 420) is preferred to `pkg_resources.declare_namespace`. See https://setuptools.pypa.io/en/latest/references/keywords.html#keyword-namespace-packages
  declare_namespace(pkg)
```